src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BindingBuildTest.cs

```diff
@@ -227,7 +227,10 @@ public void BindingCustomJavaApplicationClass ()
 				IsRelease = true,
 			};
 			binding.AndroidClassParser = "class-parse";
-			binding.Jars.Add (new AndroidItem.EmbeddedJar (() => "$(MonoDroidInstallDirectory)\\lib\\mandroid\\android-support-multidex.jar"));
+			var multidexJar = Environment.OSVersion.Platform == PlatformID.Win32NT
+				? Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.ProgramFilesX86), "MSBuild", "Xamarin", "Android", "android-support-multidex.jar") 
```

Comment:
I'd rather not hardcode `%ProgramFiles(x86)%\...`, because *eventually* we'll want to be able to build and run unit tests *on Windows*, and that environment will *not* be in `%ProgramFiles(x86)%`...

Perhaps we should go with this for now, and then fix it when we address PR #253?